towncrier/newsfragments/3095.bugfix

```diff
@@ -0,0 +1 @@
+Ensure that the form validator related to forwarding addresses allows for uppercase
```

Comment:
Filename should be 3096

Reply:
https://mailu.io/2.0/contributors/workflow.html#changelog says that it should be the ticket number... why do you want it to be the PR number?